app/helpers/declarable_helper.rb

```diff
@@ -1,6 +1,14 @@
 module DeclarableHelper
   def classification_description(commodity)
-    descriptions = [commodity.heading, commodity.ancestors, commodity]
+    tree_items = if commodity.heading?
```

Comment:
```suggestion
    tree_items = if goods_nomenclature.heading?
```

Reply:
Or declarable

Reply:
So most of what's being asked here is a bit greedy on the goods_nomenclature model. I wonder if tree_items should be pushed down to the individual model?

Reply:
e.g. Heading#tree_items Commodity#tree_items

Reply:
Then you don't have to know about whether the declarable is a heading